SDCTA/src/pages/SignUpDetails.tsx

```diff
@@ -0,0 +1,9 @@
+import { DetailsPage } from "../components/auth/DetailsPage";
+
+export const SignUpDetails: React.FC = () => {
+  return (
+    <div>
+      <DetailsPage />
```

Comment:
We might not need this wrapping, but we can just rename DetailsPage to SignUpDetailsPage or something to make it less general and clear that its associated with the SignUpDetails